### DISCUSSION TITLE
what's  the best way to change vcpkg install path name when `_` in repo's name

### DISCUSSION THREAD

**Discussion by nocanstillbb** [microsoft/vcpkg#35117] [Q&A]
i migrate some 3thPath repos to our private vcpkg registry, some repo's name contains `_`  ,but vcpkg's rule not allow port's name contains '_',  so i  named port name with '-' .  project name still use `_` .  it is works. but i get two folder in install path like this:
```bash
../
share/
| rc-genicam-api/
| | usage
| | vcpkg.spdx.json
| | vcpkg_abi_info.txt
| rc_genicam_api/
| | net_perf_check.sh*
| | package.xml
[2] /d/source/repos/dv_app_solution/vcpkg_installed/x64-windows/share/NetrwTreeListing  [netrw]  utf-8   unix  enH  
```

i want to know is there way modify install path from `rc-genicam-api` to `rc_genicam_api` ,  because  i don't want to modify those 3thpart repos.
or
can i chage `vcpkg_abi_info.json` and `vcpkg_spdx.json` generate path?

**Comment by Osyotr** [comment]
You are not forced to put everything in one dir. If a package expects some files in a path with underscores, that's fine, you can have both.

**Comment by dg0yt** [comment]
> can i change `vcpkg_abi_info.json` and `vcpkg_spdx.json` generate path?

No. This path must be `share/<PORT>`.

**Comment by nocanstillbb** [comment]
i can accept this  

thanks
